src/fighter_state.rs

```diff
@@ -535,6 +579,7 @@ fn attacking(
 
         // Do a forward jump thing
         //TODO: Fix hacky way to get a forward jump
+        //should moving and any attack that requires movement be a non exclusive state?
```

Comment:
Not 100% sure I understand what this means, but my gut reaction is that movement made during an attack should be manged by an exclusive attack state most of the time. Unless you can literally walk while attacking without the attack interfering with the walking movement at all.

Reply:
Comment should have read "additive state" but I honestly don't really remember what I was getting at with this one either.

Reply:
`non-exclusive` sounds nice too. :thinking: I could almost go with either terminology. :)